models/uniswap/arbitrum/uniswap_arbitrum_trades.sql

```diff
@@ -0,0 +1,33 @@
+{{ config(
+        alias ='arbitrum_trades'
```

Comment:
the ```alias``` can  simply be ```trades``` as the schema is defined in the dbt_project file, which will show ```arbitrum``` in the name (once you apply the changes i mentioned in other comment)

Reply:
Gotcha.  It wouldn't compile which is why I changed it initially, but it compiled OK after changing the dbt_project file.